src/views/page.html.njk

```diff
@@ -438,7 +533,7 @@ <h1>cookiemonster</h1>
                 acc[checkbox.id] = checkbox.checked;
                 return acc;
             }, {});
-        const newCard = await fetch('/check', {
```

Comment:
the rename is kinda confusing here... I'm ok with having `response` as an intermediate variable but it should be the response from the server, not the card HTML element.